### DISCUSSION TITLE
Avatar groups + hover state?

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3928] [General] [answered]
Hi!

Wondering if it's accessible to add hover states within a table within a hover state 😅:

![Screenshot 2024-06-04 at 2 48 19 PM](https://github.com/twilio-labs/paste/assets/87141551/ab83d80c-5e3e-4202-a830-982870ae5925)

We need a way to show multiple connections (sometimes multiple of the same extension type) in a table, with the ability to hover (or otherwise see) the specific name.

The idea is that the extensions are glanceable, so the avatar group makes sense to me, but I understand by adding the hover/tooltip (?) into the mix, we might be overcomplicating this component. 

Would love your thoughts, thanks!

**Comment by serifluous** [comment, marked as answer]
Hey @ceyerkes ! Do you mean a hover state on the Avatars in an Avatar Group?

By default, when you hover on an Avatar, its `name` appears (not through the Paste tooltip but the browser): https://paste.twilio.design/components/avatar#accessibility. You can see it working on the Avatar Group example here: https://paste.twilio.design/components/avatar#default-avatar-group

Tooltips need to have a Button trigger, which means the trigger will have a mouse pointer hand, which might not be what you want either.

If the connection name is important enough to show, it's probably worth considering showing the text next to the Avatar if you can.

**Reply by ceyerkes** [reply]
Thank you @serifluous ! For some reason i can't actually see the avatar group hover behavior in the docs, but i trust you 😀

**Reply by serifluous** [reply]
:sad-trombone: haha here it is. fwiw it's not as immediate as a Paste tooltip

![Screenshot 2024-06-12 at 2 38 20 PM](https://github.com/twilio-labs/paste/assets/3821840/99e94eaf-03f7-46d4-82c2-b699be51dc5a)

**Reply by ceyerkes** [reply]
perfect, thank youuuuuu!!
